content/kubernetes/release-notes/k8s-6-4-2-4.md

```diff
@@ -0,0 +1,179 @@
+---
+Title: Redis Enterprise for Kubernetes release notes 6.4.2-4 (March 2023)
+linktitle: 6.4.2-4 (March 2023)
+description: 
+weight: 57
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [ 
+    /kubernetes/release-notes/k8s-6-4-2-4.md,
+    /kubernetes/release-notes/k8s-6-4-2-4/,   ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.4.2-4 release supports Redis Enterprise Software 6.4.2 and contains new features and feature improvements.
+
+The key features, bug fixes, and known limitations are described below.
+
+## Images
+
+* **Redis Enterprise**: `redislabs/redis:6.4.2-43`
+* **Operator**: `redislabs/operator:6.4.2-4`
+* **Services Rigger**: `redislabs/k8s-controller:6.4.2-4`
+
+### OpenShift images
+
+* **Redis Enterprise**: `registry.connect.redhat.com/redislabs/redis-enterprise:6.4.2-43.rhel8-openshift`
+    (or `redislabs/redis-enterprise:6.4.2-43.rhel7-openshift` if upgrading from RHEL 7)
+* **Operator**: `registry.connect.redhat.com/redislabs/redis-enterprise-operator:6.4.2-4`s
+* **Services Rigger**: `registry.connect.redhat.com/redislabs/services-manager:6.4.2-4`
+
+### OpenShift OLM bundles
+
+* Redis Enterprise operator bundle version: `v6.4.2-4.0`
+
+## Upgrade to 6.4.2-4
+
+This release uses a new `ValidatingWebhookConfiguration` resource that replaces the old webhook resource. To use the 6.4.2-4 release, delete the old webhook resource and apply the new file.
+
+1. Delete the existing `ValidatingWebhookConfiguration` on the Kubernetes cluster (named `redb-admission`).
+
+    ```sh
+    kubectl delete ValidatingWebhookConfiguration redb-admission
+    ```
+
+1. Apply the resource from the new file.
+
+    ```sh
+    kubectl apply -f deploy/admission/webhook.yaml
+    ```
+
+## New features
+
+* Cluster-level LDAP configuration support in REC (RED-83533)
```

Comment:
Please check Rachel's latest RN for formatting and apply the same for consistency. Doesn't have to be for this push. Update later if you can.